portfolio/src/main/webapp/comments.js

```diff
@@ -0,0 +1,44 @@
+function displayComments(){
+    fetch('/comments-data')
+    .catch(error => console.log('failed to fetch comments data from server'))
```

Comment:
Maybe it is also a good idea to log the error was to help with debugging. Same below.